Source/WebCore/css/parser/CSSParserImpl.cpp

```diff
@@ -1553,11 +1589,14 @@ bool CSSParserImpl::consumeDeclaration(CSSParserTokenRange range, StyleRuleType
     if (!isExposed(propertyID, &m_context.propertySettings))
         propertyID = CSSPropertyInvalid;
 
-    if (propertyID == CSSPropertyInvalid && CSSVariableParser::isValidVariableName(token)) {
+    // @position-try doesn't allow custom properties.
+    // TODO: maybe make this logic more elegant?
+    if (propertyID == CSSPropertyInvalid && CSSVariableParser::isValidVariableName(token) && ruleType != StyleRuleType::PositionTry) {
         AtomString variableName = token.value().toAtomString();
         consumeCustomPropertyValue(range, variableName, important);
     }
 
+    // FIXME: might have to add StyleRuleType::PositionTry here?
```

Comment:
You can't use !important in position-try from what I can understand, so probably makes sense to me to add it in there.

Reply:
I added it there.